src/main/java/ultoi/util/DateTime.java

```diff
@@ -0,0 +1,56 @@
+package ultoi.util;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a data and time.
+ *
+ * @author snoidetx
+ * @version 0.0
+ */
+public class DateTime {
+    private final LocalDate date;
+    private final int time;
+
+    /**
+     * Creates a new DateTime object.
+     *
+     * @param datetime Input string to be processed.
+     * @throws UltoiException If the input string does not follow the format "YYYY-MM-DD tttt".
+     */
+    public DateTime(String dateTime) throws UltoiException {
+        String[] dateTimes = dateTime.split(" ");
+
+        try {
+            this.date = LocalDate.parse(dateTimes[0]);
+            this.time = Integer.parseInt(dateTimes[1]);
+        } catch (Exception e) {
+            throw UltoiException.wrongDateTimeFormatException();
+        }
+
+        if (this.time < 0 || time % 100 > 59 || time / 100 > 23) {
```

Comment:
Maybe you could consider using constants for 59 and 23 to point out what the numbers mean?